script/custom_build_functions.sh

```diff
@@ -1,5 +1,10 @@
 function run_cukes {
-  bin/rake acceptance --trace
+  if is_mri_192_plus; then
+    bin/rake acceptance --trace
+    return $?
+  else
+    return 0
```

Comment:
Might be worth echoing a warning explaining that this part of the build is being skipped and why.